dicom_test.go

```diff
@@ -29,8 +27,13 @@ func TestAllFiles(t *testing.T) {
 	names, err := dir.Readdirnames(0)
 	require.NoError(t, err)
 	for _, name := range names {
-		t.Logf("Reading %s", name)
-		_ = mustReadFile("examples/"+name, dicom.ReadOptions{})
+		// TODO: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

Comment:
wait, how come we hit a limit here? We are hardly pushing to this repository. Please open a ticket and place in backlog.